server/src/config.ts

```diff
@@ -78,7 +78,7 @@ export default {
   runPeriodicExportTests: isTrue(process.env.RUN_PERIODIC_EXPORT_TESTS) as boolean,
   shouldUseTranslationAPI: setGoogleApplicationCredentials() as boolean,
   staticFilesAdminPort: parseInt(process.env.STATIC_FILES_ADMIN_PORT || '8080', 10) as number,
-  staticFilesClientPort: parseInt(process.env.STATIC_FILES_CLIENT_PORT || '8080', 10) as number,
+  staticFilesParticipationPort: parseInt(process.env.STATIC_FILES_PARTICIPATION_PORT || '8080', 10) as number,
```

Comment:
In matching the pattern elsewhere so that the env files don't explicitly need to be used, this should probably back up explicitly to `process.env.STATIC_FILES_PORT`.